add-on/chrome/bootstrap.js

```diff
@@ -1298,6 +1300,10 @@ function startup(data) {
     return;
   }
 
+  // register about: handlers
+  AboutLoop.conversation.register(); // eslint-disable-line no-undef
+  AboutLoop.panel.register(); // eslint-disable-line no-undef
```

Comment:
Rather than doing the `eslint-disable-line`, lets just add AboutLoop to the `add-on/chrome/.eslintrc` like we've done with our other modules.